wpilibNewCommands/src/main/native/include/frc2/command/CommandScheduler.h

```diff
@@ -191,6 +192,12 @@ class CommandScheduler final : public wpi::Sendable,
   void UnregisterSubsystem(std::initializer_list<Subsystem*> subsystems);
   void UnregisterSubsystem(std::span<Subsystem* const> subsystems);
 
+  /**
+   * @brief Get all registered subsytems as an immutable vector
+   *
+   */
+  const std::vector<Subsystem*> GetAllRegisteredSubsystems();
```

Comment:
Placing this method between UnregisterSubsystem and UnregisterAllSubsystems feels odd, since it's interrupting the logical grouping.